packages/ui/src/components/icon/components/pin-icon.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Icon } from '../icon.component.js';
+import { type IconProps } from '../icon.types.js';
+
+export function PinIcon({
+  look = 'filled',
+  'aria-label': ariaLabel = 'Pin',
+  copyrightYear = '2023',
```

Comment:
Can we update the copyrightYear to 2025 across all components?

Reply:
updated for all icons/logos